src/modules/PowerStressModule.cpp

```diff
@@ -114,7 +115,7 @@ int32_t PowerStressModule::runOnce()
                 break;
             case meshtastic_PowerStressMessage_Opcode_CPU_FULLON: {
                 uint32_t start_msec = millis();
```

Comment:
start_msec is set to millis() ... this looks strange

Reply:
I thought so too. I think the intent here is to hamster wheel the CPU perhaps